cdm/core/src/test/java/ucar/nc2/ft2/coverage/TestCoverageCoordAxis1D.java

```diff
@@ -0,0 +1,52 @@
+package ucar.nc2.ft2.coverage;
+
+import org.junit.Test;
+import ucar.ma2.DataType;
+import ucar.nc2.constants.AxisType;
+import ucar.nc2.time.CalendarDate;
+import ucar.nc2.time.CalendarPeriod;
+import ucar.nc2.util.Optional;
+import static com.google.common.truth.Truth.assertThat;
+
+
+public class TestCoverageCoordAxis1D {
+
+  private static final CalendarDate testDate = CalendarDate.present();
+
+  private final int timeDelta = 2;
+
+  @Test
+  public void TestSubsetPresentTime1D() {
+    SubsetParams presentSubsetParams = new SubsetParams();
+    presentSubsetParams.setTimePresent();
+
+    CoverageCoordAxis1D timeAxis = create1DTimeAxis();
+    Optional<CoverageCoordAxis> subsetTimeAxis = timeAxis.subset(presentSubsetParams);
+
+    assertThat(subsetTimeAxis.isPresent()).isTrue();
+
+    if (!subsetTimeAxis.isPresent()) {
+      return;
+    }
+
+    assertThat(subsetTimeAxis.get().startValue).isEqualTo(timeDelta);
+    assertThat(subsetTimeAxis.get().endValue).isEqualTo(timeDelta);
+    assertThat(subsetTimeAxis.get().isSubset()).isTrue();
+  }
+
+  private CoverageCoordAxis1D create1DTimeAxis() {
+    CalendarDate refTime = testDate.subtract(CalendarPeriod.of(timeDelta, CalendarPeriod.Field.Day));
+    String timeUnit = "Day since " + refTime;
+
+    int valuesLen = 10;
+    double[] values = new double[valuesLen];
+    for (int i = 0; i < valuesLen; i++) {
+      values[i] = i + 1;
+    }
```

Comment:
Maybe simpler to just do
```suggestion
    double[] values = new double[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```